static/js/containers/ProfilePage.js

```diff
@@ -24,6 +24,7 @@ class ProfilePage extends React.Component<*, ProfileContainerProps, *> {
   componentDidMount() {
     const { scrollToError } = this.props;
     scrollToError.call(this);
+    document.title = "Personal Profile";
```

Comment:
this is title for 3 profile tabs, personal, eduction, work, each tab has its own title